src/GdbCommandHandler.cc

```diff
@@ -94,6 +94,7 @@ class RRCmd(gdb.Command):
         for arg in args:
             argStr += ":" + gdb_escape(arg)
         rv = gdb.execute(cmd_prefix + argStr, to_string=True);
+        gdb.execute("flushregs");
```

Comment:
I was thinking about it but we need to have add attributes for commands (or hardcode this in). I though it might of been premature optimization at this point given how rarely commands were used and we don't really know this is costly. But I don't mind doing it now.